lib/generator.js

```diff
@@ -252,6 +255,10 @@ class Generator {
   async generateFromString(asyncapiString, parserOptions = {}) {
     if (!asyncapiString || typeof asyncapiString !== 'string') throw new Error('Parameter "asyncapiString" must be a non-empty string.');
 
+    if (Object.keys(this.mapBaseUrlToFolder).length > 0) {
+      parserOptions.resolve = {resolver: this.getMapBaseUrlToFolderResolver(this.mapBaseUrlToFolder)};
```

Comment:
I wonder how this would conflict if we use both `mapBaseUrlToFolder` and the `parserOptions` [here](https://github.com/asyncapi/generator/blob/master/docs/api.md#Generator+generateFromString).

Reply:
There is a conflict, if `generateFromString` is called with a valid `resolve` configuration. Then `resolve` is overwritten by `mapBaseUrlToFolder`. I will recheck. Maybe I can move the code to `generateFromFile` and `generateFromURL`.

Reply:
Conflict is solved.